src/chrome/content/rules/WhistleOut.xml

```diff
@@ -1,18 +1,29 @@
-<ruleset name="WhistleOut (partial)">
+<ruleset name="WhistleOut">
 
 	<target host="whistleout.com" />
-	<target host="*.whistleout.com" />
+	<target host="www.whistleout.com" />
+	<target host="androidcentral.whistleout.com" />
+	<target host="androidpit.whistleout.com" />
+	<target host="comparecom.whistleout.com" />
+	<target host="gizmag.whistleout.com" />
+	<target host="help.whistleout.com" />
+	<target host="moneytalksnews.whistleout.com" />
+	<target host="r1.whistleout.com" />
+	<target host="r2.whistleout.com" />
+	<target host="r3.whistleout.com" />
+	<target host="res3.whistleout.com" />
+	<target host="techradar.whistleout.com" />
+	<target host="telegraph.whistleout.com" />
+	<target host="test.whistleout.com" />
+	<target host="trustedreviews.whistleout.com" />
+
 	<target host="whistleout.com.au" />
 	<target host="www.whistleout.com.au" />
+
 	<target host="whistleout.co.uk" />
 	<target host="www.whistleout.co.uk" />
 
```

Comment:
I'm not securing cookies without extensive tests. That's one of the major causes of issues.